packages/rehype-stringify/index.d.ts

```diff
@@ -4,6 +4,6 @@ import type {Root} from 'hast'
 import type {Plugin} from 'unified'
 // Note: defining all nodes here, such as with `Root | Element | ...` seems
 // to trip TS up.
-const rehypeStringify: Plugin<[Options] | [], Root, string>
+export const rehypeStringify: Plugin<[Options] | [], Root, string>
```

Comment:
This isn't exported by the js https://github.com/rehypejs/rehype/blob/8ddc4b84b1c38bd38bb26d8893af65495f60f2c3/packages/rehype-stringify/index.js#L3
It may make sense to use declare instead

Reply:
Sure, updated